platform/spec/services/connections/create_service_spec.rb

```
@@ -0,0 +1,62 @@
+
+      expect(connection).to have_attributes(
+        workspace: workspace,
+        source: source_connector,
+        destination: destination_connector,
+        name: "#{source_connector.name} Connection",
+        schedule_type: 'manual',
+        status: 'created',
+        sync_frequency: 'hourly',
+        namespace: 'system_defined'
+      )
+    end
```

rubocop

[convention] Example has too many lines. [12/10]

(RSpec/ExampleLength)